docs/source/map.json

```
@@ -336,6 +336,20 @@
                 "name": "University of Surrey",
                 "url": "https://www.studyacrossthepond.com/sites/default/files/University%20of%20Surrey%20logo%202107x780px.png"
             }
+        },
+        {
+            "type": "Feature",
+            "geometry": {
+                "type": "Point",
+                "coordinates": [
+                    36.11139,
+                    140.10389
+                ]
```

Flip the coordinates
```suggestion
                "coordinates": [
                    140.10389,
                    36.11139
                ]
```